R/calcualte_age.R

```diff
@@ -21,78 +26,88 @@
 #'
 #' @examples
 #' age <- calculate_age(
-#'   data          = readRDS(system.file("extdata", "test_df.RDS",
-#'                                       package = "cleanepi")),
+#'   data = readRDS(system.file("extdata", "test_df.RDS",
+#'     package = "cleanepi"
+#'   )),
 #'   target_column = "dateOfBirth",
-#'   end_date      = Sys.Date(),
-#'   age_in        = "months",
-#'   na_strings    = "-99"
+#'   end_date = Sys.Date(),
+#'   age_in = "months",
```

Comment:
A more natural example?
```suggestion
#'   age_in = "years",
```

Reply:
I guess there's no harm in adding another example but I'll leave that to your discretion.

Reply:
In turn I leave it to the package manager's discretetion :)

Reply:
I will build add an example to illustrate this